pkg/plugins/file/condition.go

```diff
@@ -8,23 +8,40 @@ import (
 
 	"github.com/sirupsen/logrus"
 
+	"github.com/updatecli/updatecli/pkg/core/result"
 	"github.com/updatecli/updatecli/pkg/core/scm"
 	"github.com/updatecli/updatecli/pkg/core/text"
 )
 
 // Condition test if a file content match the content provided via configuration.
 // If the configuration doesn't specify a value then it fall back to the source output
 func (f *File) Condition(source string) (bool, error) {
-	return f.condition(source)
+	return f.checkCondition(source)
 }
 
 // ConditionFromSCM test if a file content from SCM match the content provided via configuration.
 // If the configuration doesn't specify a value then it fall back to the source output
 func (f *File) ConditionFromSCM(source string, scm scm.Scm) (bool, error) {
 	if !filepath.IsAbs(f.spec.File) {
 		f.spec.File = filepath.Join(scm.GetDirectory(), f.spec.File)
+		logrus.Debugf("Relative path detected: changing to absolute path from SCM: %q", f.spec.File)
 	}
-	return f.condition(source)
+	return f.checkCondition(source)
+}
+
+func (f *File) checkCondition(source string) (bool, error) {
+	passing, err := f.condition(source)
+	if err != nil {
+		logrus.Infof("%s Condition on file %q errored", result.FAILURE, f.spec.File)
+	} else {
+		if passing {
+			logrus.Infof("%s Condition on file %q passed", result.SUCCESS, f.spec.File)
+		} else {
+			logrus.Infof("%s Condition on file %q did not pass", result.FAILURE, f.spec.File)
```

Comment:
What would you think to use a different message than failiure? 

```suggestion
			logrus.Infof("%s Condition on file %q did not pass", result.ATTENTION, f.spec.File)
```
"\u26A0"

Reply:
It feels weird to use « attention » since it is the symbol used when a target changes a file. Might be something to reconsider overall: choosing 2 symbols for failure and not passing, and 2 other symbols for « success » and « changed »

Reply:
But the fact of sharing the same error symbol is why I took care of having 2 really different text messages to easily discern failure from condition not passing. Maybe we could remove the « cross » from failure since it generate a clear error message in red.